zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/AuthorizationCheckBehavior.java

```diff
@@ -165,6 +182,27 @@ private Stream<String> getUserAuthorizedResourceIdentifiers(
         Stream.concat(roleAuthorizedResourceIdentifiers, groupAuthorizedResourceIdentifiers));
   }
 
+  private Stream<String> getMappingsAuthorizedResourceIdentifiers(
+      final AuthorizationRequest request) {
+    return extractUserTokenClaims(request.getCommand())
+        .<PersistedMapping>mapMulti(
+            (claim, stream) ->
+                mappingState.get(claim.claimName(), claim.claimValue()).ifPresent(stream))
```

Comment:
💭 This is just a thought. In the token there might be multiple claims, we are going to try to fetch all of them from state